docs/site/tutorials/custom_differentiation.ipynb

```diff
@@ -507,9 +507,9 @@
    "outputs": [],
    "source": [
     "/// Given a differentiable function, returns the same differentiable function except when\n",
-    "/// derivatives of this function is being computed, values in the original function that are needed\n",
-    "/// for computing the derivatives will be recomputed, instead of being captured by the differential\n",
-    "/// or pullback.\n",
+    "/// derivatives of this function are being computed. In this case, values in the original function \n",
```

Comment:
Minor suggestions:
```
In that case, values in the original function needed for computing the derivatives will be recomputed, instead of being captured by the differential or pullback.
```